docs/governance/index.md

```diff
@@ -36,3 +36,110 @@ All proposals can be found on the Hummingbot Foundation Snapshot: https://snapsh
+All Pull Requests targeting the development branch will be merged into the main code-base only if there is an approved Pull Request Proposal on the Hummingbot Snapshot page related to it.
+
+## Hummingbot Development Cycle
+The Hummingbot Client Repository has three main branches related to the development cycle of each new version:
+
+## 1.	Development branch
+All Pull Requests aiming to be included on the master branch must be targeted to the development branch. They are then promoted from development to the staging branch before passing to master
+Pull Requests targeted to the development branch will only be considered to merge into staging only when there is an approved Pull Request Proposal related to it. 
+
+## 2.	Staging branch
+
+Staging branch is used by the Foundation team to test all code changes before adding them to the master branch. The development branch is the prior step to staging branch.
+
+
+## 3.	 Master branch
+Master is the main release branch and contains the latest stable version of the Hummingbot Client. The staging branch is the step prior to master for Foundation QA review
+
+ 
+## Pull Requests Proposals Approval and Implementation Cycle
+ 
+ There is a workflow of approvals for the Pull Request ProcessÑ
+
+ ![](/assets/img/pull request proposal workflow.png)
+
+In any given month, each Hummingbot Foundation Proposal Release will have a two Months cycle:
+
+![](/assets/img/pull request proposal period.png)
+
+Month 1
+During the first month of the development cycle, Pull Request Proposals approved during this period will be scheduled to be added to the next month's release.
+Only Pull Request Proposals approved before the cut-off date will be added to the following release.
+
+Month 2
+During the second month of the development cycle, The Foundation team will merge all the remaining approved Pull Request Proposals, and move the development branch to the staging branch, and run all tests needed.
+A new release can happen on any day of the second month, as soon as all approved Pull Requests are merged, and the Foundation QA team signs off the release.
+
+## How to have your Pull Request added to the Hummingbot code base 
+
+## 1. Create the Pull Request
+For a Pull Request to be considered "open for voting", all the requirements below must be fulfilled:
+a.	Is targeted to the development branch;
+b.	Contains a clear description of what is the objective of the Pull Request;
+c.	Passes all the existing Unit Tests;
+d.	No merging conflicts exists;
+
+When all the requirements above are met, a Pull Request Proposal can be created, allowing the community to vote if the Pull Request will be merged into the development branch. If the Pull Request or the Proposal does not meet the requirements described above, it will be deleted by the Foundation Team
+
```

Comment:
Add:
"NOTE: To test and improve the process of voting and merging the approved PRP, until January 30rd only Hummingbot Foundation members will be able to create new PRPs"